blueprint-manager/src/main/resources/blueprints/configurations/druid/rds.handlebars

```diff
@@ -9,7 +9,7 @@
   }{{{#if rds.druid}}},
   "druid-common": {
     "properties": {
-      "druid.metadata.storage.type": "postgres",
+      "druid.metadata.storage.type": "postgresql",
```

Comment:
The `postgresql` could be get from the RDS view by using `{{{ rds.customrds.subprotocol }}}`.
It's confusing from druid side, because the storage type should be postgres, but this is my opininon.